### PR TITLE
Il93/mobile display heart rate

### PR DESCRIPTION
**Pull request by rafael-ramirez-orte**
# Description

Please include a short summary of the change and which issue is fixed.

### Screenshots, Videos, Gifs.
WhatsApp Image 2023-01-25 at 12 35 40 PM

Add some screenshots, videos, gifs, etc, that may help the reviewer to get a better context of the requested changes.

# Checklist:

- [ ] Pull request owner assigned
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have pulled / merged main/develop into this branch

[IL-ID]

### CODE REVIEW

File: page/DailyForm.jsx [hunk 1]
```diff
@@ -1,396 +1,385 @@
...
-// eslint-disable-next-line react/prop-types
 const DailyForm = () => {
-    const navigation = useNavigation()
-    const { control, handleSubmit, setValue } = useForm()
-    const [selectedValue, setSelectedValue] = useState('temperature')
+	const { control, handleSubmit, setValue, watch } = useForm();
+	const [ selectedValue, setSelectedValue] = useState("36");
+	const { postEvatForm } = useFetch();
+	const { heartRateTable, breathingRateTable, resultRateLevel } = EvatAlgorithm;
+	console.log(watch("FC"))
```
Comment (JavierRuvalcaba) [inline page/DailyForm.jsx]: Remove console log

File: page/DailyForm.jsx [hunk 2]
```diff
@@ -1,396 +1,385 @@
...
+		setValue("neuro", "");
+		setValue("cardio", "");
+		setValue("resp", "");
+		setValue("nurseConcern", "");
+		setValue("familyConcern", "");
+	};
 
-    const { postEvatForm } = useFetch()
+	const onSubmit = async data => {
+		console.log(data, selectedValue);
```
Comment (JavierRuvalcaba) [inline page/DailyForm.jsx]: Remove console log

File: page/DailyForm.jsx [hunk 3]
```diff
@@ -1,396 +1,385 @@
...
+					}
+				}
+			]);
+		} catch (err) {
+			console.log(err);
+		}
+		cleanForm();
+	};
+    useEffect(()=>{
+        console.log(control._fields.resp,watch("resp"))
```
Comment (JavierRuvalcaba) [inline page/DailyForm.jsx]: remove console log

File: page/DailyForm.jsx [hunk 4]
```diff
@@ -1,396 +1,385 @@
...
-                    name="familyConcern"
-                />
-                <TouchableOpacity
-                    onPress={handleSubmit(onSubmit)}
-                    style={styles.button}
-                >
-                    <Text style={styles.textbtn}>Guardar</Text>
-                </TouchableOpacity>
+    useEffect(()=>{
+        console.log(control._fields.resp,watch("cardio"))
```
Comment (JavierRuvalcaba) [inline page/DailyForm.jsx]: remove console log

File: page/DailyForm.jsx [hunk 5]
```diff
@@ -1,396 +1,385 @@
...
+                 
+				<Controller
+					control={control}
+					rules={{
+						required: true,
+					}}
+					render={({ field: { onChange, onBlur, value } }) => (
+						<TextInput
+							keyboardType="numeric"
+							placeholder="Preocupacion enfermera"
```
Comment (JavierRuvalcaba) [inline page/DailyForm.jsx]:
```suggestion
							placeholder="Preocupación enfermera"
```

File: page/DailyForm.jsx [hunk 6]
```diff
@@ -1,396 +1,385 @@
...
+				/>
+				<Controller
+					control={control}
+					rules={{
+						required: true,
+					}}
+					render={({ field: { onChange, onBlur, value } }) => (
+						<TextInput
+							keyboardType="numeric"
+							placeholder="Preocupacion familia"
```
Comment (JavierRuvalcaba) [inline page/DailyForm.jsx]:
```suggestion
							placeholder="Preocupación familia"
```

**Review by JavierRuvalcaba — CHANGES_REQUESTED** [review]

**Review by JavierRuvalcaba — APPROVED** [review]

**Review by pedro123-67we — APPROVED** [review]
muy bien!!!

**Pull request merged**